dapps/shop/backend/routes/affiliate.js

```diff
@@ -24,6 +27,48 @@ function authAffiliate(req, res, next) {
   next()
 }
 
+/**
+ * Formats a BigQuery product row into a dshop product obj
+ */
+function bqProductFormatter(product) {
+  // product ID is only in the IPFS path returned
```

Comment:
Can't we use the new product_id col from BQ?

Reply:
No.  

Nick/FE is expecting the product ID that has already been used.  Here's an example I was given to emulate: https://affiliate.ogn.app/affiliate/products.json  

Though I do realize now that I forgot to prepend listing ID to it, which I'll fix shortly.